### PR TITLE
Add "Godot at BIG Festival 2023" article

### PR DESCRIPTION
**Pull request by vnen**
Add article about the presence of Godot at BIG Festival 2023.

Also added my own avatar.

### CODE REVIEW

File: collections/_article/godot-at-big-festival-2023.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+categories: ["news"]
+author: George Marques
+date: 2023-06-15 00:00:00
+
+image: /storage/blog/covers/big-2023.webp
+---
+
+Following our first time [hosting a booth at GDC](https://godotengine.org/article/gdc-2023-retrospective/) earlier this year, Godot will also have a booth at the [BIG Festival](https://bigfestival.com.br/o-festival/) this year. BIG will happen between June 28th and July 2nd, so it is right around the corner.
+
+For the unaware, BIG Festival is a huge event for indie game developers that happens yearly in Brazil since 2012. It is the biggest event of this kind in Latin America and. BIG brings out many big players in the industry, not only from Brazil but also from the rest of the world. It has a good amount space for showcasing games, awards in multiple categories, as well as talks about a multitude of topics.
```
Comment (Calinou) [inline collections/_article/godot-at-big-festival-2023.md]: > It is the biggest event of this kind in **Latin America and. BIG brings out**

Is a word missing (or is the `and` extraneous)?

Comment (vnen) [reply collections/_article/godot-at-big-festival-2023.md]: Yeah, the `and` shoudlnn't be there.

File: collections/_article/godot-at-big-festival-2023.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+BIG Festival is a huge event for indie game developers that happens yearly in Brazil since 2012. It is one of the biggest in Latin America and gathers out many big players in the industry, not only from Brazil but also from the rest of the world. It has a good amount space for showcasing games, an awards ceremony, and industry-related talks.
+
+During GDC we met some of the organizers of the BIG Festival which started a conversation that ended up with our presence being confirmed at the event. Our booth is being kindly sponsored by [W4 Games](https://w4games.com/). Without the support of sponsors like them, our presence at the event would not have been possible.
+
+We will also have a talk about the current state and plans for Godot's growth, presented by one of the project's co-founders: Ariel Manzur. It is scheduled to happen on Thursday, **June 29th** at **18:35** (local time), in **Stage 1**. You can see the details on the official [program page](https://event.bigfestival.com.br/talks/11/516.)
```
Comment (adamscott) [inline collections/_article/godot-at-big-festival-2023.md]: The current _program page_ link is broken.

```suggestion
We will also have a talk about the current state and plans for Godot's growth, presented by one of the project's co-founders: Ariel Manzur. It is scheduled to happen on Thursday, **June 29th** at **18:35** (local time), in **Stage 1**. You can see the details on the official [program page](https://event.bigfestival.com.br/talks/11/516).
```

Comment (vnen) [reply collections/_article/godot-at-big-festival-2023.md]: Thanks!

**Pull request merged**